bootstrap/017_query_history.sql

```diff
@@ -166,3 +170,18 @@ BEGIN
         call internal.set_config(:key, 'true');
     end if;
 END;
+
+create or replace procedure internal.update_qtag_day()
+returns number
+language sql
+comment = 'materialize qtag_filter for a single day until fully backfilled'
+as
+begin
+update internal_reporting_mv.query_history_complete_and_daily set qtag_filter=tools.qtag_to_map(qtag)
+where start_time::date = (select max(start_time::date) from internal_reporting_mv.query_history_complete_and_daily where qtag_filter is null and qtag is not null) and qtag_filter is null and qtag is not null;
+let updates number := (select $1 from table(result_scan(last_query_id())));
+update internal_reporting_mv.query_history_complete_and_daily_incomplete set qtag_filter=tools.qtag_to_map(qtag)
+where start_time::date = (select max(start_time::date) from internal_reporting_mv.query_history_complete_and_daily_incomplete where qtag_filter is null and qtag is not null) and qtag_filter is null and qtag is not null;
+updates := (select $1 from table(result_scan(last_query_id()))) + :updates;
+return updates;
+end;
```

Comment:
should add an `EXCEPTION` block here in case something goes awry since there isn't one in the caller.

Reply:
added